networking/ipv6.md

```diff
@@ -1,145 +1,134 @@
 
-| Flag | Value/Content |
-| ---- | ------------- |
-| `--address` | Should be set to the appropriate IPv6 address or `::` for all IPv6 addresses. |
-| `--cluster-dns` | Should be set to the IPv6 address that will be used for the service DNS, this must be in the range used for `--service-cluster-ip-range`. |
-| `--node-ip` | Should be set to the IPv6 address of the node. |
+**Optional**
+- [Configure the Kubernetes control plane to operate over IPv6](#configure-the-kubernetes-control-plane-to-operate-over-ipv6)
 
-##### kube-proxy
+#### Enable dual stack, Kubernetes
 
-| Flag | Value/Content |
-| ---- | ------------- |
-| `--bind-address` | Should be set to the appropriate IPv6 address or `::` for all IPv6 addresses on the host. |
-| `--master` | Should be set with the IPv6 address where the kube-apiserver can be accessed. |
-| `--cluster-cidr` | Should be set to match the {{site.prodname}} IPv6 IPPool. |
+1. Set up a new cluster following the Kubernetes {% include open-new-window.html text='prerequisites' url='https://kubernetes.io/docs/concepts/services-networking/dual-stack/#prerequisites' %} and {% include open-new-window.html text='enablement steps' url='https://kubernetes.io/docs/concepts/services-networking/dual-stack/#enable-ipv4-ipv6-dual-stack' %}.
 
-### Enabling IPv6 support in {{site.prodname}}
+1. Using the [{{site.prodname}} Kubernetes install guide]({{site.baseurl}}/getting-started/kubernetes/self-managed-onprem/onpremises), download the right {{site.prodname}} manifest for the cluster, and datastore type. 
 
-To enable IPv6 support when installing {{site.prodname}} follow the
-steps below.
+1. Edit the CNI config (`calico-config` ConfigMap in the manifest), and enable IP address allocation by setting both modes to true.
 
-1. Download the {{site.prodname}} manifest you wish to update for IPv6
-   deployment and save it as `calico.yaml`.
-1. If the ipam section in the `cni_network_config` in the `calico.yaml` file
-   has `"type": "calico-ipam"` then it should be modified to
-   [disable IPv4 assignments and enable IPv6
-   assigments](/reference/cni-plugin/configuration#ipam).
-1. Add the following environment variable to the calico-node DaemonSet in
-   the `calico.yaml` file.
-   
-   ```yaml
-   - name: IP6
-     value: "autodetect"
    ```
-1. For clusters **not** provisioned with kubeadm (see note below), also add the following environment variable to the calico-node DaemonSet in
-   the `calico.yaml` file. Be sure to set the value for `CALICO_IPV6POOL_CIDR`
-   to the desired pool, it should match the `--cluster-cidr` passed to the
-   kube-controller-manager and to kube-proxy.
+       "ipam": {
+           "type": "calico-ipam",
+           "assign_ipv4": "true",
+           "assign_ipv6": "true"
+       },
+   ```
+
+1. Configure IPv6 **IP address autodetection** and **IP pool** by adding the following variable settings to the environment for the `calico-node` container:
```

Comment:
I think we've gone into too much detail here, which (a) is partly unnecessary and (b) still doesn't cover the FELIX_IPV6SUPPORT.  I suggest: "Configure IPv6 support and the default IPv6 IP pool by adding ..."

Reply:
Good edit. I'm loathe to make these types of edits without supervision. Sometimes the details are important, sometimes not.